docs/en/academic/zkn-method.md

```diff
@@ -32,6 +32,9 @@ An internal link is written with the syntax of `[[This is the link]]`. If you `C
 
 The second function triggered by such a link is a global search inside your currently selected directory. It will merely take the link contents, place it in your search field and automatically "press Enter", to initiate the search. This way you can not only open exact files, but also find all other files that link to the file you just opened. So a link in the format `[[<your-id>]]` would open that specific file and also search for all files that link back to this file.
 
+> **Tip**: It is possible to utilize the standard markdown linking format to set a title for an internal link. For example, `[My text]([[<my-doc-id>]])` will render as if it was a regular link. When you click on it once, it will enter edit mode and reveal the internal link. Then, you can `Ctrl + click` the internal link to trigger internal link functionality. This 'hack' has a caveat, though: when exporting your document, this may create empty links (depending on your export preferences).
+![](../img/internal_linking_tip.gif)
+
```

Comment:
> It actually generates empty links for [Empty link text](). Zettlr does not render [Empty link text]() as a link.

Right, if I'm not mistaken Zettlr requires an actual link in the braces, but it may be that Pandoc still attempts to render something. But it doesn't break anything, right?

Reply:
No, it doesn't break anything